cli/cmd/utils.go

```diff
@@ -141,7 +142,31 @@ func doesKubeconfigExist(*cobra.Command, []string) error {
 }
 
 func getLokoConfig() (*config.HCLConfig, hcl.Diagnostics) {
-	return config.LoadConfig(viper.GetString("lokocfg"), viper.GetString("lokocfg-vars"))
+	lokocfgFiles, diags := config.LoadHCLFiles(viper.GetString("lokocfg"), "lokocfg")
+	if diags.HasErrors() {
+		return nil, diags
+	}
+
+	exists, err := util.PathExists(viper.GetString("lokocfg-vars"))
```

Comment:
If we don't check this and just try to load it, it should be fine.

Reply:
Actually, we should only check if the file exists if it has been explicitly specified in the CLI. Because if user is explicit about the file and it does not exist, that should raise an error. But if we read the "default" config files and glob finds none, I think it's fine to proceed and just fail later, that the components are not configured etc. WDYT?

Reply:
in `cli/cmd/root.go`

```
RootCmd.PersistentFlags().String("lokocfg", "./", "Path to lokocfg directory or file")
viper.BindPFlag("lokocfg", RootCmd.PersistentFlags().Lookup("lokocfg"))
RootCmd.PersistentFlags().String("lokocfg-vars", "./lokocfg.vars", "Path to lokocfg.vars file")
viper.BindPFlag("lokocfg-vars", RootCmd.PersistentFlags().Lookup("lokocfg-vars"))
```

How to differentiate if the cli flag was passed or its the default.

Reply:
Maybe we should remove the default value from the CLI and move it somewhere else?  But perhaps this is out of scope then.